src/environments/environment.preprod.ts

```diff
@@ -11,4 +11,5 @@ export const environment = {
   sentryTracesSampleRate: 1.0,
   sentryReplaysSessionSampleRate: 0,
   sentryReplaysOnErrorSampleRate: 1.0,
+  sentryEnvironment: 'production',
```

Comment:
The sentry environment is set to 'production' but it is the 'preprod' environment file